lib/std/os/windows/bits.zig

```diff
@@ -24,6 +24,11 @@ pub const STD_OUTPUT_HANDLE = maxInt(DWORD) - 11 + 1;
 /// The standard error device. Initially, this is the active console screen buffer, CONOUT$.
 pub const STD_ERROR_HANDLE = maxInt(DWORD) - 12 + 1;
 
+pub const WINAPI: builtin.CallingConvention = if (builtin.arch == .i386)
+    .Stdcall
+else
+    .C;
+
```

Comment:
What? Why? stdcall is always used except on arm

Reply:
No, see the current code: https://github.com/ziglang/zig/blob/eb33394d14e29600d36280b6797de0d8f40076c1/src/stage1/codegen.cpp#L165-L168

stdcall doesn't exist on x86_64.

Reply:
Oh right! M$VC ignores even cdecl on x86_64 and uses its own 'x64' callconv.